plugins/CircuitViewer/io/MorphologyLoader.cpp

```diff
@@ -702,4 +704,32 @@ PropertyMap MorphologyLoader::getCLIProperties()
     pm.setProperty(PROP_USE_SDF_GEOMETRIES);
     return pm;
 }
+
+void createMissingMaterials(Model& model)
```

Comment:
that's still nuclear bomb to kill a fly :) The materials that are required to be created should be known in here, but @hernando said he will look into this.

Reply:
I would make it a different PR.

Reply:
Let me figure it out, I was the last one touching that code.